dydx/dydxPresenters/dydxPresenters/_v4/Receipt/dydxTradeReceiptPresenter.swift

```diff
@@ -80,10 +87,43 @@ final class dydxTradeReceiptPresenter: dydxReceiptPresenter {
 
     private func updateExpectedPrice(tradeSummary: TradeInputSummary?, market: PerpetualMarket) {
         let value = dydxFormatter.shared.dollar(number: tradeSummary?.price?.doubleValue, digits: market.configs?.displayTickSizeDecimals?.intValue ?? 0)
-        expectedPriceViewModel.title = DataLocalizer.localize(path: "APP.GENERAL.PRICE")
+        expectedPriceViewModel.title = DataLocalizer.localize(path: "APP.TRADE.EXPECTED_PRICE")
         expectedPriceViewModel.value = value
     }
 
+    private func updateLiquidationPrice(position: SubaccountPosition?, market: PerpetualMarket) {
+        let title = DataLocalizer.localize(path: "APP.TRADE.LIQUIDATION_PRICE_SHORT")
+        let unit = AmountTextModel.Unit.dollar
+        let tickSize = market.configs?.displayTickSizeDecimals?.intValue.asNsNumber
+        liquidationPriceViewModel.title = title
+        liquidationPriceViewModel.value = createAmountChangeViewModel(title: title, tradeState: position?.liquidationPrice, tickSize: tickSize, unit: unit)
+    }
+
+    private func updatePositionMargin(position: SubaccountPosition?) {
+        let title = DataLocalizer.localize(path: "APP.TRADE.POSITION_MARGIN")
+        let unit = AmountTextModel.Unit.percentage
+        positionMarginViewModel.title = title
+        positionMarginViewModel.value = createAmountChangeViewModel(title: title, tradeState: position?.marginUsage, tickSize: 2, unit: unit)
```

Comment:
Web shows Margin as the dollar value.  Let me add it to Abacus and we can change this later.

Reply:
will handle next